common/dynamicconfig/constants.go

```diff
@@ -84,6 +84,11 @@ var (
 		9000,
 		`VisibilityPersistenceMaxWriteQPS is the max QPC system host can query visibility DB for write.`,
 	)
+	VisibilityPersistenceSlowQueryThreshold = NewGlobalDurationSetting(
```

Comment:
Can we do per namespace?

Reply:
I don't see a lot of motivation to, but I may be missing something. What would be useful about per namespace?

Reply:
Hm, I'm not sure if there will be any use case tbh. We can change later if we need.